changes/34780-scep-proxy-retries

```diff
@@ -0,0 +1,2 @@
+- Fixed issue where MDM profile retry limits were interfering with Smallstep SCEP proxy renewal
+  attempts, particularly in cases of expired SCEP challenges.
```

Comment:
See https://github.com/fleetdm/fleet/issues/35527#issuecomment-3517921857 for a list of other potential fixes/improvements to be addressed in follow up sprints.